src/com/facebook/buck/go/GoSymlinkTree.java

```diff
@@ -39,10 +40,9 @@
 
   protected GoSymlinkTree(
```

Comment:
SymlinkTree used SourcePaths, which I thought were generally from source files. In this case all the entries are build rule outputs. Is it still encouraged to use SymlinkTree with BuildTargetSourcePath in that case?